src/app/globalStatusBar/login/stub/login.js

```diff
@@ -0,0 +1,61 @@
+import submissionParse from '../../../submissions/stub/submission.js'
+
+/**
+ * @name constructRequestBody
+ * @desc Create an URLSearchParam object suitable to be sent as log in request
+ * @param {String} username - Username
+ * @param {String} password - Password
+ * @return {URLSearchParams} `new URLSearchParam()` with given params
+ */
+
+function constructRequestBody(username, password) {
+    let out = new URLSearchParams()
+    out.append('username', username)
+    out.append('password', password)
+    return out
+}
+
+/**
+ * @name login
+ * @desc Log in function. On completion, window.hestia.loggedIn will have the result of the log in attempt
+ * @param {String} username - Username
+ * @param {String} password - Password
+ * @param {Function} func - Function to call when window.hestia.updateLoginState returned.
+ * @return {Promise} - a Promise that resolves to the return value of func()
+ * @author minhducsun2002
+ */
+
+async function login(username, password, func) {
+    return (
+        fetch(
+            `http://${window.location.hostname}:${window.location.port}/login`,
```

Comment:
Shorten this to "/api/login"

Reply:
Address in https://github.com/vnma0/hestia/pull/19/commits/84c0282afababde0e3ad70cf90345d947d22553c.